amenity-points.mss

```diff
@@ -146,9 +146,11 @@
     marker-clip: false;
   }
 
-  [feature = 'tourism_chalet'][zoom >= 17] {
+// Ircama (revised zoom for tourism_chalet; allowed it at lower zoom, but with reduced size)
+  [feature = 'tourism_chalet'][zoom >= 13] {
     point-file: url('symbols/chalet.p.16.png');
     point-placement: interior;
+    [zoom <= 15] { point-transform:"scale(0.9)"; } // ircama: this reduces the size of the symbol for lower zooms
```

Comment:
Can you please suggest a different way than <=?
[zoom < 16]
or
[zoom >= 13][zoom < 16]?

Meanwhile I would also add tourism_wilderness_hut, represented by now with the same symbol and modality of amenity=shelter and visible when zoom >= 13